package.json

```diff
@@ -102,12 +112,12 @@
     "flow": "flow",
     "flow-typed-rebuild": "rm -rf flow-typed/ && flow-typed install && flow-typed install --rootDir packages/@cruise-automation/button && flow-typed install --rootDir packages/@cruise-automation/tooltip && flow-typed install --rootDir packages/react-key-listener && flow-typed install --rootDir packages/regl-worldview",
     "docs": "webpack-dev-server",
-    "docs-deploy": "gh-pages -d docs/public",
+    "docs-deploy": "cp -r docs/public __temp_deploy__ && cp -f packages/webviz-core/public/index.html __temp_deploy__/index.html && cp -f packages/webviz-core/node_modules/wasm-lz4/wasm-lz4.wasm __temp_deploy__/dist/wasm-lz4.wasm && gh-pages -d __temp_deploy__ && rm -rf __temp_deploy__",
```

Comment:
This now includes using a temporary directory and "resolving" the symlinks to actual files.